File: codeforces/52.buy_a_shovel/buy_a_shovel.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int k, r, n, d;
  cin >> k >> r;
  n = 1;
  while (true) {
    d = ((n * k) % 10);
    if ((d == r) || (d == 0)) {
      break;
    }
    n++;
  }
  cout << n << endl;
  return 0;
}

File: codeforces/52.buy_a_shovel/buy_a_shovel.py
# Author: José Rodolfo (jric2002)
k, r = list(map(int, input().split(" ")))
n = 1
while (True):
  d = ((n * k) % 10)
  if ((d == r) or (d == 0)):
    break
  n += 1
print(n)